### ISSUE TITLE
Irys SDK (Arweave Upload)

### ISSUE THREAD

**Issue by IkigaiLabsETH** [IkigaiLabsETH/ikigailabs.xyz#159]
**Arweave Uploads**

HELIUS DOCS: You can choose to provide your own off-chain metadata via the uri field. If no uri is specified, we will build a JSON file and upload to Arweave on your behalf. The file will adhere to the [v1.0 Metaplex JSON standard](https://docs.metaplex.com/programs/token-metadata/changelog/v1.0). We upload to Arweave via [Irys](https://docs.irys.xyz/) (formerly known as Bundlr).

//

Irys started out as a solution to both scale Arweave and reduce pain points experienced by developers. Our horizontal and linear scaling solutions radically increased transactions per second, from 9 TPS on Arweave directly to more than [50,000 TPS(opens in a new tab)](https://www.youtube.com/watch?v=JKEivHKDXAo&t=1s) when using Irys.

Data uploaded to Irys is given a millisecond-accurate timestamp, attributes and authorship details before being passed to Arweave for permanent storage. This information is used to create a signed [receipt](https://docs.irys.xyz/learn/receipts) that can be used to [verify](https://docs.irys.xyz/developer-docs/irys-sdk/api/utils-verifyReceipt) the data's provenance at any time.

//

TLDR: We probably do not need to use the IRYS SDK as it's already used by the HELIUS API but wanted to clarify what it is , wand why it matters. There may be some edge use cases (upload biz docs) where this sdk is useful later down the line
